app/interpreters/dashboards/panel_interpreter.rb

```diff
@@ -23,6 +24,12 @@ def interpret_panel_links(panel)
         end
       end
 
+      def interpret_panel_geospatial_map(panel)
+        return unless panel["options"].present? && panel["viewType"] == 'geospatial'
+
+        panel["options"]["geospatial"]["geoJsonUrl"] = ENV["GEO_JSON_URL"]
```

Comment:
@hengsokly is there any way to validate, does the URL is a valid format? s.th like that, what do you think?